### PR TITLE
fix(manager-pci-common): update peer dependencies

### PR DESCRIPTION
**Pull request by aboungnaseng-ovhcloud**
ref: MANAGER-14622

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix 14822
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

Update peer dependencies in pci common, manager react components, react shell client

### CODE REVIEW

File: packages/manager-react-components/package.json [hunk 1]
```diff
@@ -46,17 +44,19 @@
...
+    "react-i18next": "^14.0.5",
     "sass": "1.71.0",
     "tailwindcss": "^3.4.4",
     "uuid": "^9.0.1"
   },
   "devDependencies": {
     "@babel/core": "7.22.10",
     "@mdx-js/react": "^3.0.1",
+    "@ovh-ux/manager-core-api": "^0.9.0-alpha.0",
+    "@ovh-ux/manager-react-shell-client": "^0.8.0-alpha.0",
```
Comment (frenautvh) [inline packages/manager-react-components/package.json:57]:
```suggestion
```
it should be removed from devDependencies

Comment (anooparveti) [reply packages/manager-react-components/package.json:57]: @frenautvh it is required for successful "tsc" https://github.com/ezolenko/rollup-plugin-typescript2/issues/277#issuecomment-1112504639

Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/package.json:57]: whats can be the solutionto not duplicate theses dependencies as we already have "prepare": "tsc" command in "@ovh-ux/manager-core-api" and "@ovh-ux/manager-react-shell-client" ?

Comment (aboungnaseng-ovhcloud) [reply packages/manager-react-components/package.json:57]: or in pci common we dont have prepare comand

**Review by anooparveti — APPROVED** [review]

**Review by chipp972 — APPROVED** [review]

**Pull request merged**